test/org/sagebionetworks/bridge/services/ScheduledActivityServiceRecurringTest.java

```diff
@@ -199,16 +195,16 @@ public void persistedActivitiesAreFilteredByEndsOn() throws Exception {
         assertTrue(activities2.size() < activities.size());
     }
     
-    private ScheduleContext getContextWith2DayWindow(DateTimeZone zone) {
-        return getContext(zone, DateTime.now(zone).plusDays(2));
+    private ScheduleContext getContextWith2DayWindow(DateTimeZone requestZone) {
+        return getContext(EST, DateTime.now(requestZone).plusDays(2));
     }
     
-    private ScheduleContext getContext(DateTimeZone zone, DateTime endsOn) {
+    private ScheduleContext getContext(DateTimeZone persistedZone, DateTime endsOn) {
         // Setting the endsOn value to the end of the day, as we do in the controller.
         return new ScheduleContext.Builder()
             .withStudyIdentifier(study.getStudyIdentifier())
             .withClientInfo(ClientInfo.UNKNOWN_CLIENT)
-            .withTimeZone(zone)
+            .withTimeZone(persistedZone)
             .withAccountCreatedOn(DateTime.now())
             // Setting the endsOn value to the end of the day, as we do in the controller.
             .withEndsOn(endsOn.withHourOfDay(23).withMinuteOfHour(59).withSecondOfMinute(59))
```

Comment:
I've rewritten this test to use NOW instead of DateTimeUtils functions, and I'm now setting the NOW value in the schedule context, and removed all uses of DateTime.now(). I believe this should make the tests stable whenever they are run (we shouldn't rely on system time at all in the scheduler).